resources/lang/en/filament-logger.php

```diff
@@ -25,4 +25,7 @@
     'resource.label.new' => 'New',
     'resource.label.old_value' => 'Old Value',
     'resource.label.new_value' => 'New Value',
+    'resource.label.hint' => 'Can be key or value',
```

Comment:
`resource.label.hint` is not specific enough

Reply:
I know. What do you suggest? I just tried to make it similar to the previous keys.

Reply:
`properties_hint` perharps?

Reply:
Fine by me. I have updated the translation key: https://github.com/Z3d0X/filament-logger/pull/54/commits/c9f1da5cb47b0531d41ef924d2fc6ef78a98f795